docs/users-guide/09-multi-series-charting.md

```diff
@@ -7,73 +7,79 @@ Data in isolation is rarely all that useful. One of the best ways to add context
 - New users per day vs. returning users per day.
 - Orders per day from a few different product lines.
 
-### In Metabase there are two main ways to get data side-by-side.
+### Two ways to display data side-by-side
```

Comment:
```suggestion
### Displaying data side-by-side
```

Reply:
There's actually a third thing you can do, which is adding a second metric to your question. E.g., you can do a Sum of Subtotal and a Count of Rows by Created At on the Orders table.

You can also do a join in Custom Question mode, where you join e.g. count of rows by created at with a different time series saved question, like Users Over Time.